### PR TITLE
Fix no method errors

### PR DESCRIPTION
**Pull request by kookster**
Seeing errors in production where getting the images, addresses, and stories related to accounts are throwing errors.

The controllers were still assuming these were has many rather than has one resources, and trying to retrieve and link to them incorrectly.

### CODE REVIEW

File: test/representers/api/account_representer_test.rb [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 describe Api::AccountRepresenter do
 
-  let(:account)     { build_stubbed(:account) }
+  let(:account) { create(:account) }
```
Comment (chrisrhoden) [inline test/representers/api/account_representer_test.rb]: Why was this change necessary? Is the representer writing to the database?

Comment (kookster) [inline test/representers/api/account_representer_test.rb]: to generate nested urls in the representers, we need the account id

Comment (chrisrhoden) [inline test/representers/api/account_representer_test.rb]: rgr.

**Comment by kookster** [comment]
@chrisrhoden anything else I should do for this?
